### PR TITLE
Eduhelx

### PR DESCRIPTION
**Pull request by Acharya-Saurya**
Added eduhelx brand to files (similar to cmas).

### CODE REVIEW

File: appstore/core/templates/base.html [hunk 1]
```diff
@@ -54,6 +56,9 @@
             {% elif APPLICATION_BRAND == "heal" %}
             <a href="/"><img class="navbar-brand navbar-brand-heal" id="img-brand-heal-logo" src="{% static "images/heal/heal-social-logo.png" %}"
                              alt="HEAL Logo"></a>
+            {% elif APPLICATION_BRAND == "eduhelx" %}
+            <a href="/"><img class="navbar-brand" id="img-brand-eduhelx-logo" src="{% static "images/heal/heal-social-logo.png" %}" <!--TBD -->
```
Comment (ismagilani) [inline appstore/core/templates/base.html:60]: reference brand directory here i.e. images/eduhelx/logo-name.png

File: appstore/appstore/settings/eduhelx_settings.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+from product.configuration import ProductSettings, ProductColorScheme, ProductLink
+
+# TODO remove Application brand once the new frontend is complete and
+# the django templates in core are removed.
+APPLICATION_BRAND = "eduhelx"
+
+PRODUCT_SETTINGS = ProductSettings(
+    brand="eduhelx",
+    title="EduHeLx",
+    logo_url="/static/images/cmas/cmas-logo.png", #TBD
```
Comment (ismagilani) [inline appstore/appstore/settings/eduhelx_settings.py:11]: use an eduhelx directory under static/images

File: appstore/core/templates/base.html (hunk 1 above)
Comment (ismagilani) [inline appstore/core/templates/base.html:60]: use images/eduhelx/ and eduhelx logo which I will request from marketing

File: appstore/core/views.py [hunk 3]
```diff
@@ -334,6 +334,10 @@ def get_brand_details(brand):
         "commonsshare": {
             "name": "CommonsShare",
             "logo": "logo-lg.png"
+        },
+        "eduhelx": {
+            "name": "EduHeLx",
+            "logo": "heal-social-logo.png" #TBD
```
Comment (ismagilani) [inline appstore/core/views.py:340]: need correct logo reference

**Review by ismagilani — CHANGES_REQUESTED** [review]

**Comment by ismagilani** [comment]
@Acharya-Saurya please send an email to Marcus Anderson and ask him to design a logo for EduHeLx instance.  Tell him that it is for HeLx platform and for a data science course at UNC.  He may have one we can already use.  Please CC me on the email.

**Comment by Acharya-Saurya** [comment]
Recreating eduhelx branch

**Pull request closed without merging**